docs/nx-cloud/features/dynamic-agents.md

```diff
@@ -18,26 +18,51 @@ jobs:
+You can configure Nx Cloud to execute a different number of agents based on the size of your PR's affected changes. Define any number of **changesets** (the number of agents and types of agents) to use for different sized PRs and pass them in a configuration file to your `start-ci-run` command.
 
-Create a file called `dynamic-changesets.yaml` in the `.nx/workflows` directory of your repo.
+Start by creating a file called `distribution-config.yaml` in the `.nx/workflows` directory of your repo. This file will contain a `distribute-on` property that will be used to define the changesets to use for your PR. You can name your changesets anything you want.
 
-```yaml {% fileName=".nx/workflows/dynamic-changesets.yaml" %}
+{% callout type="warning" title="The order of your changesets matters!" %}
+Define your changesets in order of increasing size (i.e. smallest changesets are defined before larger changesets). Nx Cloud uses the position of your changesets as part of its calculations to dynamically determine the correct changeset to use for your PR.
+{% /callout %}
+
+```yaml {% fileName=".nx/workflows/distribution-config.yaml" %}
 distribute-on:
   small-changeset: 3 linux-medium-js
   medium-changeset: 6 linux-medium-js
   large-changeset: 10 linux-medium-js
 ```
 
+You can also specify a `default` changeset if you only want one changeset to be used for all PRs. Note that `default` is a reserved keyword so do not use it if you would like to define multiple changesets.
+
+```yaml {% fileName=".nx/workflows/distribution-config.yaml" %}
+distribute-on:
+  default: 3 linux-medium-js
+```
+
+You can have as many changesets as you want. Based on the number of changesets specified, each changeset is assigned an equal percentage out of 100. Nx Cloud can determine the percentage of affected projects in your PR and use that value to evaluate which changeset to use.
+
 {% callout type="deepdive" title="How is the size of the PR determined?" %}
-To determine the size of the PR, Nx Cloud calculates the relationship between the number of [affected projects](/ci/features/affected) and the total number of projects in the workspace. It then assigns it to one of the three categories: small, medium, or large.
+Nx Cloud calculates the relationship between the number of [affected projects](/ci/features/affected) and the total number of projects in the workspace to determine the size of a PR.
 {% /callout %}
 
-You can then reference it in your CI pipeline configuration:
+## Setting up Dynamic Agents in your CI Pipeline
+
+In the example below, each changeset would be assigned an equal percentage range out of 100%. If Nx Cloud determines that 30% of your PRs have been affected, then it will use the medium changeset to distribute the workload on. If Nx Cloud determines that 55% of your PRs have been affected, it will use the large changeset.
```

Comment:
```suggestion
In the example below, each changeset would be assigned an equal percentage range out of 100%. If Nx Cloud determines that 30% of your projects have been affected, then it will use the medium changeset to distribute the workload on. If Nx Cloud determines that 55% of your projects have been affected, it will use the large changeset.
```